src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -190,20 +187,33 @@ class Breakpoints extends Component<Props> {
 
     const groupedBreakpoints = groupBy(
       sortBy([...breakpoints.valueSeq()], bp => bp.location.line),
-      bp => getBreakpointFilename(bp.source)
+      bp => bp.source.url
     );
 
     return [
       ...Object.keys(groupedBreakpoints)
-        .sort()
-        .map(filename => {
+        .sort((urlA, urlB) => urlA.split("/").pop() > urlB.split("/").pop())
+        .map(url => {
+          const split = getRawSourceURL(url).split("/");
+          const file = split[split.length - 1].split("?")[0];
+          const groupBreakpoints = groupedBreakpoints[url].filter(
+            bp => !bp.hidden && (bp.text || bp.originalText)
+          );
+
+          if (!groupBreakpoints.length) {
+            return null;
+          }
+
           return [
-            <div className="breakpoint-heading" title={filename} key={filename}>
-              {filename}
+            <div
+              className="breakpoint-heading"
+              title={url}
+              key={url}
+              onClick={() => this.props.selectSourceURL(url)}
```

Comment:
We should use `selectSource` where we can because two sources can share the same URLs.

https://github.com/pranoy10/debugger.html/blob/chevronfix/src/actions/sources/select.js#L83